client/apps/edit/components/header/index.coffee

```diff
@@ -51,8 +51,11 @@ module.exports = class EditHeader extends Backbone.View
 
   save: (e) ->
     e.preventDefault()
-    @$('#edit-save').text 'Saving...'
-    @article.trigger('finished').save()
+    @$('#edit-save').text('Saving...').removeClass('attention')
+    if @article.get('published')
+      @article.trigger('savePublished').trigger('finished')
```

Comment:
This is strange, but since the button doesn't serialize certain data (title, lead_paragraph), I triggered this to happen on the layout's client code on this line: 
https://github.com/artsy/positron/pull/308/files#diff-7662da217426d1c27c6c3c03f651708dR96